### DISCUSSION TITLE
Mancato salvataggioautomazioni dopo aggiornament HA e Zigbee2MQTT

### DISCUSSION THREAD

**Discussion by labattazza** [indomus/forum#1695] [HUB: Home Assistant]
Buon weekend a tutti, vi chiedo consiglio erpché mi è capitata una cos strana con l'aggiornamento all'ultima versione di Home Assistant.

Ho una installazione Core su Docker, e così anche per Broker MQTT Mosquitto, Zigbee2MQTT e Zwave2MQTT.

Dopo aver aggiornata alla 2022.5.3 il pannello integrazioni mi segnalava un errore per mancata compatibilità di Zwave2MQTT. Aggiorno pure lui e tutto si sistema.

Noto però che i componenti Zigbee sono irraggiungibili, così come l'interfaccia web. Però guardando i log (a quel che ricordo, era tarda sera e potrei ricordare male la scansione temporale) se premevo un pulsante vedevo il comando passare dal broker.

Il giorno dopo ho aggioranto anche Zigbe2MQTT e oplà, sia portale web che  entità ricompaiono.
Posso accendere e spegnere le luci da pannello web, ma i pulsanti che avevo associato con le automazioni non vanno.

Nel pannelo entità vedo che le automazini risultano disabilitate (probabilmente perché dopo l'aggiornamento HA non vedeva le entità). Le cancello perché nessun tentativo di ripristino" è andato a buon fine.

Le ricreo con il tool grafico ma.. dopo aver premuto il pulsante "Salva", alla pagina riassuntiva non le trovo.

Vi è mai capitata una cosa simile? Potrei prenderlo come un segno del destino per costringermi a scriverle a manina e inserirle in un apposito file .yaml separato?

Grazie mille!

**Comment by snakuzzo** [comment]
Mai capitato...
Comunque se hai bisogno di aiuto, con quello che hai scritto ci facciamo poco. Servirebbe una sfera di cristallo per capire cosa succede.
Quando chiedete aiuto dovete postare le evidenze (log, configurazioni, etc), non i comportamenti e poi chiedere se è capitato ad altri.

**Reply by labattazza** [reply]
Ciao 
hai ragione, il quesito è troppo generico probabilmente, ma non avendo nessun segnale di errore all'atto del salvataggio, non avevo proprio idea di dove poter cominciare a cercare.
Stasera provo a vedere se tra i log compare qualcosa quando creo una automazione... non avevo pensato che potrebbe elencare anche problemi che nascono in Lovelace. 
Intanto grazie!

**Reply by labattazza** [reply]
Ciao

alla fine ho capito dove stava il problema. C'era una duplicazione tra il file configuration.yaml e automations.yaml, figla di una mia errata pulizia. Non vedevo nulla nei log del frontend perché avevo settato ad "Error" e non a "warning".
